apps/payments/next/tsconfig.json

```diff
@@ -32,6 +25,13 @@
     "node_modules",
     "jest.config.ts",
     "src/**/*.spec.ts",
-    "src/**/*.test.ts"
+    "src/**/*.test.ts",
+    "**/*.stories.ts",
+    "**/*.stories.js"
```

Comment:
Are we using .ts _and_ .js Storybook stories, or was this boilerplate?

Reply:
Boilerplate.

As per your question below though, it's a good point. Does it need to have all these? cc @xlisachan remove or leave a comment?